pkg/storaged/details.js

```diff
@@ -34,7 +34,25 @@ define([
      */
 
     function init_details(client, jobs) {
-        var type, name;
+        var type, name, state;
+
+        function ensure_state_multipathd_running() {
+            if (state.multipathd_running === undefined) {
```

Comment:
Watching the service status might actually be the clean option here - everything else basically boils down to polling, which we don't really want.